kuma/users/tests/test_views.py

```diff
@@ -109,6 +110,89 @@ def test_bug_811751_banned_user(self):
         response = self.client.get(url, follow=True)
         self.assertNotEqual(response.status_code, 403)
 
+    def test_get_ban_user_view(self):
+        # For an unbanned user get the ban_user view
+        testuser = self.user_model.objects.get(username='testuser')
+        admin = self.user_model.objects.get(username='admin')
+
+        self.client.login(username='admin', password='testpass')
+        ban_url = reverse('users.ban_user',
+                          kwargs={'user_id': testuser.id})
+
+        resp = self.client.get(ban_url)
+        eq_(200, resp.status_code)
+
+        # For a banned user redirect to user detail page
+        UserBan.objects.create(user=testuser, by=admin,
+                               reason='Banned by unit test.',
+                               is_active=True)
+        resp = self.client.get(ban_url)
+        eq_(302, resp.status_code)
+        ok_(testuser.get_absolute_url() in resp['Location'])
+
+    def test_common_reasons_in_template(self):
```

Comment:
Maybe. A lot of the ``test_templates.py`` code looks like view tests to me, and won't survive the Great Test Cleanup.  I'll leave this up to @dchukhin's judgement.